samza-core/src/test/java/org/apache/samza/context/TestContextImpl.java

```diff
@@ -0,0 +1,107 @@
+package org.apache.samza.context;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
nit: don't use *

Reply:
I agree with not using * and static imports for production code, but for these test static imports (e.g. Assert, Mockito) that are used widely in unit tests to reduce verbosity, how do you feel about allowing it? IDEs automatically consolidate static imports to use *, so it is inconvenient to remember to re-expand them for all tests (although I guess there probably is some IDE setting to avoid using *). Also, there are already a lot of other tests in Samza which use * for Assert/Mockito.

Reply:
Spoke in person. It should be possible to configure the IDE to not consolidate to a wildcard, and it would be good to stick to the non-wildcard pattern for any new tests.
I'll update these tests to not use the wildcard.